onboarding_project/squad_pantry_app/migrations/0001_initial.py

```diff
@@ -0,0 +1,64 @@
+# Generated by Django 2.0 on 2018-02-03 07:45
+
+from django.conf import settings
```

Comment:
Let's have a single migration. It does not really matter for this project, but when deploying, having one large migration is faster than many small migrations